### PR TITLE
all component pages

### PR DESCRIPTION
**Pull request by lomesshh**
- In this PR I've added all component pages like Toast, Footer, Homepage, Loader, Navbar, Pagenotfound & all routes

### CODE REVIEW

File: src/components/pages/Footer.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        </div>
+        <div class="footer__social">
+          <a href="https://github.com/lomesshh">
+            <i class="fab fa-github fa-2x"></i>
+          </a>
+          <a href="https://www.linkedin.com/in/lomesshh/">
+            <i class="fab fab fa-linkedin fa-2x"></i>
+          </a>
+          <a href="https://twitter.com/lomesshh">
+            <i class="fab fa-twitter fa-2x"></i>
```
Comment (anuj-kumary) [inline src/components/pages/Footer.js]: Remove all class with className

**Review by anuj-kumary — APPROVED** [review]

**Review by anuj-kumary — COMMENTED** [review]
- Your code is very clean and you used good naming convention
Some suggestion:-
- Change class to className and give **img** tag to **alt** attributes

**Review by shrey27 — APPROVED** [review]
Overall code structure is very clean and readable.
Suggestion: You can use barrel exports for context hooks.

PR is ready to merge

**Comment by lomesshh** [comment]
> 



> Overall code structure is very clean and readable. Suggestion: You can use barrel exports for context hooks.
> 
> PR is ready to merge

Thanks for the suggestion, I'll do these changes at the end so that i can properly import them into all other files

**Comment by lomesshh** [comment]
> * Your code is very clean and you used good naming convention
>   Some suggestion:-
> * Change class to className and give **img** tag to **alt** attributes

Thanks for suggestion, i did all the changes as you suggested

File: src/components/pages/Footer.js (hunk 1 above)
Comment (lomesshh) [reply src/components/pages/Footer.js]: i did the changes

**Pull request merged**